docs/src/user-guide/quick-start/overview.md

```diff
@@ -0,0 +1,151 @@
+# Overview
+
+This page will guide you through CLP's system requirements and help you choose which flavour of CLP will be best for you.
+
+If you already have CLP set up, click your flavour of CLP below to learn how to use it.
+
+::::{grid} 1 1 2 2
+:gutter: 2
+
+:::{grid-item-card}
+:link: clp-json
+CLP for JSON logs
+^^^
+Learn about compressing and searching JSON logs.
+:::
+
+:::{grid-item-card}
+:link: clp-text
+CLP for unstructured text logs
+^^^^^^^^^^^^
+Learn about compressing and searching unstructured text logs.
+:::
+::::
+
+---
+
+## System Requirements
+
+To run properly on your system, CLP requires a few other programs. Before you set up CLP, ensure that you have the following installed.
+
+### Docker
+
+CLP uses Docker to deploy its different components. You can check whether Docker is already installed on your system by running the command
+
+```bash
+docker version
+```
+
+If Docker is not on your system, follow the instructions [here][Docker] to install it.
+
+If you're not running as root, ensure Docker can be run [without superuser privileges][docker-non-root].
+
+If you're using Docker Desktop, ensure version 4.34 or higher is installed, and [host networking is enabled][docker-desktop-host-networking].
```

Comment:
```suggestion
NOTE:

* If you're not running as root, ensure Docker can be run [without superuser privileges][docker-non-root].
* If you're using Docker Desktop, ensure version 4.34 or higher is installed, and [host networking is enabled][docker-desktop-host-networking].
```

* I find lists are easier to read rather than independent lines of text.
* I'm suggesting just a plain "NOTE:" rather than a `{note}` admonition since I think readers have a tendency to skip the latter when they're rushing.